wallet/src/wallet.rs

```diff
@@ -134,7 +134,7 @@ pub trait CapeWalletExt<'a, Backend: CapeWalletBackend<'a> + Sync + 'a> {
         // it.
         cap_asset: AssetDefinition,
         dst_addr: UserAddress,
-        amount: u64,
+        amount: Amount,
```

Comment:
To be consistent with the Seahorse API,
* use `RecordAmount`
* Allow conversions, e.g. `amount: impl Into<RecordAmount>`

Reply:
Same goes for all the `Amount` parameters in this file